modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -1547,17 +1556,31 @@ impl Wallet {
         cfg: &WalletConfig,
         task_group: &TaskGroup,
         bitcoind: &DynBitcoindRpc,
-    ) -> (watch::Receiver<Option<u32>>, watch::Receiver<Feerate>) {
+    ) -> anyhow::Result<(watch::Receiver<Option<u32>>, watch::Receiver<Feerate>)> {
         let (block_count_tx, block_count_rx) = watch::channel(None);
         let (fee_rate_tx, fee_rate_rx) = watch::channel(cfg.consensus.default_fee);
 
+        let sources = std::env::var(FM_WALLET_FEERATE_SOURCES_ENV)
+            .unwrap_or_else(|_| match cfg.consensus.network.0 {
+                Network::Bitcoin => "https://mempool.space/api/v1/fees/recommended#.;https://blockstream.info/api/fee-estimates#.%221%22".to_owned(),
+                _ => String::new(),
+            })
+            .split(';')
+            .filter(|s| !s.is_empty())
+            .map(|s| Ok(Box::new(FetchJson::from_str(s)?) as Box<dyn FeeRateSource>))
+            .chain(iter::once(Ok(
+                Box::new(bitcoind.clone()) as Box<dyn FeeRateSource>
+            )))
+            .collect::<anyhow::Result<Vec<Box<dyn FeeRateSource>>>>()?;
+        let feerates = Arc::new(std::sync::Mutex::new(vec![None; sources.len()]));
+
         let mut desired_interval = tokio::time::interval(if is_running_in_test_env() {
             // In devimint, the setup is blocked by detecting block height changes,
             // and polling more often is not an issue.
             debug!(target: LOG_MODULE_WALLET, "Running in devimint, using fast node polling");
             Duration::from_millis(100)
         } else {
-            Duration::from_secs(10)
+            Duration::from_secs(30)
```

Comment:
We might need to query for block height and fee rate on different interverals, if this is a problem.